From 20e7d932008a6adf6a137e7865e5420a65712c27 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sergio=20Migu=C3=A9ns=20Iglesias?= <sergio@lony.xyz>
Date: Wed, 6 Oct 2021 12:28:20 +0200
Subject: [PATCH] Laptop config

---
 config.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config.h b/config.h
index cf6a5db..cc243e4 100644
--- a/config.h
+++ b/config.h
@@ -41,7 +41,7 @@ static int attachdirection = 3;
 /* TODO make the screeCount dynamic so that DWM doesn't have to be recompiled
  * every time it's executed on a different system.
  */
-static const int screenCount = 2;
+static const int screenCount = 1;
 
 /*
  * XF86 Keys. very useful for laptops.
@@ -54,7 +54,7 @@ static const int screenCount = 2;
 
 /* tagging */
 /* TODO Make tags dynamic. */
-static char *tags[] = { "1", "2", "3", "4" };
+static char *tags[] = { "1", "2", "3", "4", "5", "6", "7", "8" };
 
 
 static const Rule rules[] = {
-- 
2.33.0

